### DISCUSSION TITLE
I installed ChromeOs on my Laptop and Play store apps and Linux beta is not even there

### DISCUSSION THREAD

**Discussion by ComputerJerks** [sebanc/brunch#1154] [Q&A]
Linux (betra) option  not there in settings.

I searched in google and found no solution.

Please help!!

Then afterwards I read that we need MBR support. 

I have installed ChromeOS on MBR and in legacy system.

So is it necessary to do anything or am I unlucky!!

Is there a solution.

Can I add MBR patch after installation of ChromeOS

Or should I install ChromeOS again and can you give a detailed explanation

where I should  place the MBR patch in Linuxmint System I mean where in Bootable Drive?

**Comment by tejasraman** [comment]
What is your laptop model and CPU? UEFI is always preferred, and if your laptop is 2012> (or a Mac) then it's usually already on UEFI - some models shipped with Win7 and need a BIOS setting change
